src/librbd/librbd.cc

```diff
@@ -2753,6 +2779,26 @@ extern "C" int rbd_mirror_image_get_status(rbd_image_t image,
   return 0;
 }
 
+extern "C" int rbd_stat_watch(rbd_image_t image, uint64_t *handle,
+			      rbd_callback_t watch_cb, void *arg)
+{
+  librbd::ImageCtx *ictx = (librbd::ImageCtx *)image;
+  C_StatWatchCB *wctx = new C_StatWatchCB(watch_cb, arg);
```

Comment:
Ah, it is a missing bit yet. I will fix it. Thanks. Initially I thought the watch context would be deleted automatically on unregister, but later decided that it is up to caller to delete it but forgot to update this part.